charts/admission-controller/README.tpl

```diff
@@ -12,365 +12,92 @@ $ pre-commit run -a
 
 -->
 
-# Admission Controller
-
-[{{ .Project.Name }}]({{ .Project.URL }}) features ActivityAudit and ImageScanning on a Kubernetes Cluster.
-<br/>{{ .Project.Description }}
-
-## TL;DR;
-
-```
-$ helm repo add {{ .Repository.Name }} {{ .Repository.URL }}
-$ helm repo update
-$ helm upgrade --install sysdig-{{ .Release.Name }} {{ .Repository.Name }}/{{ .Chart.Name }} \
-      --create-namespace -n {{ .Release.Namespace }}{{ with .Chart.Version }} --version={{.}} {{ end }} \
-      --set clusterName=CLUSTER_NAME \
-      --set sysdig.secureAPIToken=SECURE_API_TOKEN
-```
-
-- [Configuration](#configuration)
-- [Usages](#usages)
-- [Confirm Working Status](#confirm-working-status)
-- [Troubleshooting](#troubleshooting)
-
-<br/><br/>
 
-## Introduction
+# Admission Controller
 
 This chart deploys {{ .Project.App }} on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
+## Overview
 
+[{{ .Project.Name }}]({{ .Project.URL }}) provides Audit Logging and optional Image Scanning capabilities to secure your Kubernetes environment.
```

Comment:
The Project.URL is pointing to https://docs.sysdig.com/en/docs/sysdig-secure/scanning/admission-controller/ which is the Legacy Scanning engine Admission COntroller in the docs. I think instead it should point to https://docs.sysdig.com/en/docs/installation/sysdig-secure/install-agent-components/kubernetes/install-kubernetes-audit-logging/?

But we don't have any page in the docs were it explains the AC provides the 2 features, linking to the corresponding page.

In my update to the public docs for the new Admission Controller I proposed something like https://sysdig-docs-pr-1348.onrender.com/en/docs/installation/sysdig-secure/install-agent-components/kubernetes/install-kubernetes-admission-controller/ as the entry page for Admission Controller installation that could work.

Reply:
Project.URL can be updated in the docs.yaml file, I am pushing changes with the proposed URL, but feel free to revert if you prefer current option.

Reply:
I have accepted your suggestion. Shall we merge?